### PR TITLE
feat: re-enable TrustyAI in downstream but set to removed (T.P)

### PR DESCRIPTION
**Pull request by zdtsw**
this is a cherry-pick of e38052db038ceba53cb3d211f185a10ef3ba5165 to re-add into RHOAI

Many thanks for submitting your Pull Request ❤️!

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/CONTRIBUTING.md)
- [ ] Pull Request contains description of the issue
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request

## JIRA issue:

ref: https://issues.redhat.com/browse/RHOAIENG-7920

## Merge criteria:

- [ ] Have a meaningful commit messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by zdtsw** [comment]
rhoai-2.12 is created and we can start review process for this PR

File: bundle/manifests/rhods-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -52,6 +52,9 @@ metadata:
               "trainingoperator": {
                 "managementState": "Removed"
               },
+              "trustyai": {
+                "managementState": "Removed"
```
Comment (VaishnaviHire) [inline bundle/manifests/rhods-operator.clusterserviceversion.yaml:56]: Should the value be set to Managed? Will the component GA in 2.12?

Comment (zdtsw) [reply bundle/manifests/rhods-operator.clusterserviceversion.yaml:56]: it will only get into RHOAI as  TP first, no plan when set to GA for now.

**Review by VaishnaviHire — APPROVED** [review]
/lgtm

**Pull request merged**